src/Services/ListenTogether/ListenTogether.Infrastructure/AzureDeveloperCliCredential.cs

```diff
@@ -0,0 +1,148 @@
+﻿using Azure.Core;
```

Comment:
Please use AzureDeveloperCliCredential from the beta version of the Azure.Identity lib.

Reply:
As per your suggestion, fix the version of "Azure.Identity" to "1.9.0-beta.2" and use AzureDeveloperCliCredential from Azure.Identity library.